pkg/kubectl/cmd/expose.go

```diff
@@ -175,6 +176,9 @@ func RunExpose(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 		}
 		params["labels"] = kubectl.MakeLabels(labels)
 	}
+	if cmdutil.GetFlagBool(cmd, "headless") {
```

Comment:
i removed `readline -f` and replaced something that works across platforms so that codecgen works fine.

`go get github.com/coreos/etcd`?! awesome!

PTAL the latest commit.